danesfield/core/models/dataset.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+from typing import List
+
+from django.db import models, transaction
+from rdoasis.algorithms.models import Dataset
+from rgd_3d.models import Mesh3D, Tiles3D
+from rgd_imagery.models import Image
+
+RGD_IMAGERY_EXTENSIONS = ('.tif', '.png')
+RGD_3D_EXTENSIONS = ('.ply', '.obj')
+
+
+class IngestedOutputFiles(models.Model):
+    dataset = models.ForeignKey(Dataset, on_delete=models.CASCADE, related_name='ingested_files')
+
+    imagery_files = models.ManyToManyField(Image, related_name='+')
+    mesh_3d_files = models.ManyToManyField(Mesh3D, related_name='+')
+    tiles_3d_files = models.ManyToManyField(Tiles3D, related_name='+')
+
+    @classmethod
+    @transaction.atomic
+    def from_dataset(cls, dataset: Dataset):
```

Comment:
This function doesn't touch the `tiles_3d_files` field at all. Is that waiting on [this](https://github.com/ResonantGeoData/ResonantGeoData/pull/618) PR?

Reply:
Technically yes it's waiting on that PR. I was able to pin the RGD django packages to that PR branch so we can use the code in the meantime, but I forgot to update this function - I'll go ahead and do that